rel/i18n/emqx_bridge_hstreamdb.hocon

```diff
@@ -47,4 +47,58 @@ NOTE: When you use `raw record` template (which means the data is not a valid JS
 record_template.label:
 """HStream Record"""
 
+action_parameters.desc:
+"""Action specific configuration."""
+
+action_parameters.label:
+"""Action"""
+
+grpc_flush_timeout.desc:
+"""Period for flushing gRPC calls to the HStreamDB server"""
+
+grpc_flush_timeout.label:
+"""gRPC Flush Period"""
+
+aggregation_pool_size.desc:
+"""Size of Record Aggregation Pool"""
+
+aggregation_pool_size.label:
+"""Aggregation Pool Size"""
+
+max_batches.desc:
+"""Maximum number of unconfirmed batches in the flush queue"""
+
+max_batches.label:
+"""Max Batches"""
+
+writer_pool_size.desc:
+"""Writer Pool Size"""
+
+writer_pool_size.label:
+"""Writer Pool Size"""
+
+batch_size.desc:
+"""Maximum number of insert data clauses that can be sent in a single request"""
+
+batch_size.label:
+"""Max Batch Append Count"""
+
+batch_interval.desc:
+"""Maximum interval in milliseconds that is allowed between two successive (batch) request"""
```

Comment:
duration() is a string like `1s` or `100ms`. there is no need to specify precision in docs.
```suggestion
"""Maximum interval that is allowed between two successive (batch) request."""
```